modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/src/main/resources/site-initializer/fragments/group/src/customer-portal/fragments/footer/index.html

```diff
@@ -0,0 +1,17 @@
+<div class="cp_footer_home">
+<div class="cp_footer_container">
```

Comment:
can you please indent the `<div>` that are inside the `<div class="cp_footer_home">`.

Reply:
Done!